erica_app/erica/pyeric/eric.py

```diff
@@ -297,6 +297,18 @@ def process_verfahren(self, xml_string, verfahren, abruf_code=None, transfer_han
         finally:
             self.close_cert_handle(cert_handle)
 
+    def check_tax_number(self, tax_number):
+        fun_check_tax_number = self.eric.EricMtPruefeSteuernummer
+        fun_check_tax_number.argtypes = [c_void_p, c_char_p]
+        fun_check_tax_number.restype = c_int
+
+        try:
+            res = fun_check_tax_number(self.eric_instance, tax_number.encode())
+            check_result(res)
+            return True
+        except EricWrongTaxNumberError:
+            return False
```

Comment:
I know what you mean and I was wondering that, too. However, I think that it leaves a lot of complicated ERiC details to the upper layers (routes in that case). Therefore, I would prefer to do it that way, especially because it is no real exception.